CustomScript/customscript.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Is `python3` purposefully there?

Reply:
This was a mistake (I was testing something else).
Removed!